modelling/src/main/java/org/axonframework/modelling/command/inspection/ForwardingMessageHandlingMember.java

```diff
@@ -0,0 +1,21 @@
+package org.axonframework.modelling.command.inspection;
+
+import org.axonframework.commandhandling.CommandMessageHandlingMember;
+import org.axonframework.messaging.Message;
+
+/**
+ * Interface describing a message handler capable of forwarding a specific command.
+ *
+ * @since 4.6
+ * @param <T> The type of entity to which the message handler will delegate the actual handling of the command
+ */
+public interface ForwardingMessageHandlingMember<T> extends CommandMessageHandlingMember<T> {
```

Comment:
I'd rename this interface to `ForwardingCommandMessageHandlingMember`.

Reply:
Or, we could remove the `extends` from this interface to make it `Message` agnostic.
That would also require moving this interface to a general package instead of `modelling.command.inspection`.

An option, but I can't conceive how we would need this functionality for other message types at the moment.